lib/filters.js

```diff
@@ -0,0 +1,66 @@
+var _ = require('lodash'),
```

Comment:
Ah yes. Left that in but I'd imagine it will be back when/if we start writing more filters. I'll remove it for now until I/we start writing more filters perhaps.